docs/CHANGELOG.md

```diff
@@ -33,6 +33,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   mainly Flaw, Affect, Tracker (OSIDB-2065)
 - Add support for additional_fields in Jira BTS (OSIDB-696)
 - Add scripts/restore_pg.sh script for restoring sql dump
+- Add support for UAT (OSIDB-2447)
```

Comment:
Wrong section, should be in `## Unreleased`

Reply:
Thank you so much !